index.html

```diff
@@ -1,11 +1,62 @@
 <!DOCTYPE html>
 <html>
+  
   <head>
     <title>Relaxr</title>
     <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i" rel="stylesheet">
   </head>
+  
   <body>
+    
     <header>
+    	<div id="title">
+    		<h1>Relaxr</h1>
+    	</div>
+    	<h2>Get piece of mind with a single tap</h2>
+    	<div id="getitnow">
+    		<h3>Get it Now</h3>
+    	</div>
     </header>
+
+    <nav>
```

Comment:
Make this a section. While it doesn't affect how the page looks semanticly its off and that could affect your SEO.